tools/generator/src/DTO/XcodeScheme+TestAction.swift

```diff
@@ -0,0 +1,5 @@
+extension XcodeScheme {
+    struct TestAction: Equatable, Decodable {
+        let targets: Set<String>
+    }
```

Comment:
These are small types, and `XcodeScheme` itself is small. Can we have these nested in the `XcodeScheme` declaration itself? Or do you see each of these files getting larger for each respective type?

Reply:
Either way is fine with me. I assume that the actions will receive additional attributes over time. However, we can cross that bridge when get to it. I will put them in one file for now.